tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -109,317 +49,309 @@ describe('Target aggregates', () => {
+        await helperFunctions.updateSettings(targetsConfig, onlineUser, contactSummaryScript);
+
+        await commonPage.goToAnalytics();
+        await targetAggregatesPage.goToTargetAggregates(true);
+
+        const expectedTargets = [
+          { id: 'a_target', title: 'what a target!', progressBar: false, counter: '58' },
+          { id: 'b_target', title: 'the most target', progressBar: true, counter: '27%' },
+        ];
+
+        const expectedContacts = helperFunctions
+          .getDocsByPlace(contactDocs, districtHospital1._id)
+          .map(contact => ({ _id: contact._id, name: contact.name, counter: 'No data', progress: 0 }));
+
+        const context = {
+          contacts: expectedContacts,
+          period: CURRENT_PERIOD,
+          isCurrentPeriod: true,
+          place: districtHospital1.name
+        };
+        const asserts = { hasMultipleFacilities: false, contactValues: false };
+
+        await helperFunctions.assertData(context, TARGET_VALUES_BY_CONTACT, expectedTargets, asserts);
+
+        await targetAggregatesPage.openTargetDetails(expectedTargets[0]);
+        await targetAggregatesPage.clickOnTargetAggregateListItem(clarissa._id);
+        // wait until contact-summary is loaded
+        expect(await contactsPage.getContactInfoName()).to.equal('Clarissa');
+        // assert that the activity card exists and has the right fields.
+        expect(await contactsPage.getContactCardTitle()).to.equal('Activity this month');
+
+        await helperFunctions.validateCardFields(['yesterday Clarissa', '40', '50%']);
+
+        await browser.back();
+
+        const firstTargetItem =
+          await (await targetAggregatesPage.getTargetItem(expectedTargets[0], CURRENT_PERIOD, districtHospital1.name));
+        await helperFunctions.assertTitle(firstTargetItem.title, expectedTargets[0].title);
+
+        await targetAggregatesPage.openTargetDetails(expectedTargets[1]);
+
+        await targetAggregatesPage.clickOnTargetAggregateListItem(prometheus._id);
+        // wait until contact-summary is loaded
+        await (await contactsPage.contactCardSelectors.contactCardName()).waitForDisplayed();
+        expect(await contactsPage.getContactInfoName()).to.equal('Prometheus');
+        // assert that the activity card exists and has the right fields.
+        expect(await contactsPage.getContactCardTitle()).to.equal('Activity this month');
+        await helperFunctions.validateCardFields(['yesterday Prometheus', '18', '15%']);
+
+        await browser.back();
+        const secondTargetItem =
+          await (await targetAggregatesPage.getTargetItem(expectedTargets[1], CURRENT_PERIOD, districtHospital1.name));
+        await helperFunctions.assertTitle(secondTargetItem.title, expectedTargets[1].title);
+      });
+    });
 
-      await commonPage.goToAnalytics();
-      await targetAggregatesPage.goToTargetAggregates(true);
+    describe('Offline user with multiple places associated', () => {
```

Comment:
Same question here. Is it the same behavior for `online` users with multiple places associated?

Reply:
Yes, you are correct. It is the same behavior for online and offline users